report_comments.md

```diff
@@ -0,0 +1,113 @@
+## Report Comments API
+With Report Comments api V4 you can:
+
+- [List Report Comments](#list-report-comments)
+- [Create a Report Comment](#create-a-report-comment)
+- [Fetch a Report Comment](#fetch-a-report-comment)
+- [Available fields](#available-report-comment-fields)
+
+
+### List report comments
+Fetch a paginated list of report comments.
+
+Optional params:
+- `report_id`: if present filter entries by report_id.
+- `team_id`: if present filter entries by team_id.
+- `user_id`: if present filter entries created by user_id.
+- `updated_after`: if present only return entries updated after given date. Valid values are dates in ISO8601 format.
+- `orderby`: if present allow specify the response order by providing one order clause
+  made of `<field_name> <direction>`. Allowed fields:
+  `updated_at`. Directions: `asc` or `desc`.
+  examples:
+    - `?orderby=updated_at+asc`
+
+```
+GET /api/v4/report_comments?updated_after=2022-05-24T19:30:30.278+12:00&orderby=updated_after+asc&fields=-report_id,report_uuid
```

Comment:
not sure about the `fields=-report_id,report_uuid` here? is the `-` correct? besides, `report_id` is not listed in the response payload